docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -11,46 +11,64 @@ other resources such as Kubernetes clusters and applications coming in the futur
 ## Installation & upgrade
 
 Head over to the [Downloads](https://goteleport.com/download/) page to download the most recent
-version of Teleport Connect. As of now, Teleport Connect supports macOS only. Double-click the
-downloaded `.dmg` file and drag the Teleport Connect icon to the Applications folder.
+version. Teleport Connect supports macOS, Linux, and Windows.
+
+<Tabs>
+<TabItem label="macOS">
```

Comment:
I think the tab items are a good solution for now.

In the future it'd probably be better to have a per platform scope like we do have for Cloud/Enterprise/OSS so that Windows users don't have to click through every `<Tabs>` component and select Windows each item.

But I'll leave that for the docs team to figure it out, maybe a need for that will arise in another area of the docs.

Reply:
I've created [an issue](https://github.com/gravitational/docs/issues/162) in the Docs repo to review this idea.